sdcm/utils/loader_utils.py

```diff
@@ -221,6 +224,7 @@ def run_post_prepare_cql_cmds(self):
             self.log.debug("Execute post prepare queries: %s", post_prepare_cql_cmds)
             self._run_cql_commands(post_prepare_cql_cmds)
 
+    @optional_stage('prepare_write')
     def run_prepare_write_cmd(self):
```

Comment:
these `run_prepare_write_cmd`s are starting nemesis when flag is set `if self.params.get('nemesis_during_prepare'):`, 
so skipping it will make nemesis thread not to be started.

Reply:
Agree. If I understand it correctly, the nemesis is 'included' into the test stage in this case - i.e. if stage is not executed the nemesis shouldn't as well.

Reply:
if someone wants a nemesis to start during prepare stage it doesn't mean he/she wants to skip it in case of skipping prepare stage.
It just specifies the moment of starting it.

Reply:
I see. The name of the parameter can be misleading then. Something like `start_nemesis_during_prepare` would be more clear.
I'll update from decorating `run_prepare_write_cmds` to using `optional_test_stages` function inside it.

Reply:
Updated the prepare_write stage handling